### DISCUSSION TITLE
Windows task scheduler ( hidden )

### DISCUSSION THREAD

**Discussion by iNfRaS0NiC** [Taxel/PlexTraktSync#1300] [General]
Hello,
is there a way to create a Windows task scheduler and which file to use? Also how to hide it from appearing every time when its running ?  
Thank you, great work!

**Comment by simonc56** [comment]
`setup.bat` does it :
https://github.com/Taxel/PlexTraktSync/blob/3bbd2477f9b6517f4c211023d1435eb09dd49906/setup.bat#L69

To hide it, you can make it run by another user account or run it through a vbs script.
https://serverfault.com/questions/9038/run-a-bat-file-in-a-scheduled-task-without-a-window
